### PR TITLE
zh: sync docs/releases/supported-releases

### PR DESCRIPTION
**Pull request by my-git9**
Please provide a description for what this PR is for.
zh: sync docs/releases/supported-releases
image

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Configuration Infrastructure
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

**Comment by windsonsea** [comment]
Could you help to sync the `line 51` too, which has a minor change:

image

**Comment by my-git9** [comment]
> Could you help to sync the `line 51` too, which has a minor change:
> 
> image
It has been synced, see:
image

**Comment by windsonsea** [comment]
> > Could you help to sync the `line 51` too, which has a minor change:
> > image
> 
> It has been synced, see: image

Of course not. It shall be `2022 年 10 月 12 日`

**Comment by my-git9** [comment]
> > > Could you help to sync the `line 51` too, which has a minor change:
> > > image
> > 
> > 
> > It has been synced, see: image
> 
> Of course not. It shall be `2022 年 10 月 12 日`

O, thanks, I was the one who neglected

File: content/zh/docs/releases/supported-releases/index.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ test: n/a
 | master          | 否，仅限开发 | - | - | - | - |
 | 1.15            | 是                   | 2022 年 8 月 31 日 | ~ 2023 年 3 月（预期）  | 1.22, 1.23, 1.24, 1.25         | 1.16, 1.17, 1.18, 1.19, 1.20, 1.21  |
 | 1.14            | 是                   | 2022 年 5 月 24 日 | ~ 2023 年 1 月（预期）  | 1.21, 1.22, 1.23, 1.24         | 1.16, 1.17, 1.18, 1.19, 1.20        |
-| 1.13            | 是                   | 2022 年 2 月 11 日 | ~ 2022 年 10 月（预期） | 1.20, 1.21, 1.22, 1.23         | 1.16, 1.17, 1.18, 1.19              |
+| 1.13            | 是                   | 2022 年 2 月 11 日 | ~ 2022 年 10 月 12 日（预期） | 1.20, 1.21, 1.22, 1.23         | 1.16, 1.17, 1.18, 1.19              |
```
Comment (windsonsea) [inline content/zh/docs/releases/supported-releases/index.md:51]: It's wrong. Please really check it by following the en version. 1.13 has been released months ago. It shall not still be `expected`!

**Review by windsonsea — CHANGES_REQUESTED** [review]

File: content/zh/docs/releases/supported-releases/index.md (hunk 1 above)
Comment (my-git9) [reply content/zh/docs/releases/supported-releases/index.md:51]: image
yes

File: content/zh/docs/releases/supported-releases/index.md [hunk 2]
```diff
@@ -48,7 +48,7 @@ test: n/a
 | master          | 否，仅限开发 | - | - | - | - |
 | 1.15            | 是                   | 2022 年 8 月 31 日 | ~ 2023 年 3 月（预期）  | 1.22, 1.23, 1.24, 1.25         | 1.16, 1.17, 1.18, 1.19, 1.20, 1.21  |
 | 1.14            | 是                   | 2022 年 5 月 24 日 | ~ 2023 年 1 月（预期）  | 1.21, 1.22, 1.23, 1.24         | 1.16, 1.17, 1.18, 1.19, 1.20        |
-| 1.13            | 是                   | 2022 年 2 月 11 日 | ~ 2022 年 10 月（预期） | 1.20, 1.21, 1.22, 1.23         | 1.16, 1.17, 1.18, 1.19              |
+| 1.13            | 是                   | 2022 年 2 月 11 日 | ~ 2022 年 10 月 12 日  | 1.20, 1.21, 1.22, 1.23         | 1.16, 1.17, 1.18, 1.19              |
```
Comment (windsonsea) [inline content/zh/docs/releases/supported-releases/index.md:51]: 大哥，就这一个小文字，改了几次了？？？？

还不对！！！咱能上点心吗？别给 dao 抹黑行吗？

❌❌❌❌

**Review by windsonsea — CHANGES_REQUESTED** [review]

**Review by windsonsea — APPROVED** [review]

**Pull request merged**